openapi/src/main/resources/templates/openworld-sdk/api.mustache

```diff
@@ -47,13 +47,22 @@ import java.util.UUID
 * {{{description}}}
 */
 {{#operations}}
-open class {{classname}}(clientConfiguration: {{#isRapid}}Rapid{{/isRapid}}ClientConfiguration = {{#isRapid}}Rapid{{/isRapid}}ClientConfiguration.EMPTY) : {{#isRapid}}RapidOpenApiStub{{/isRapid}}{{^isRapid}}OpenApiStub{{/isRapid}}(clientConfiguration) {
+open class {{classname}} private constructor(clientConfiguration: {{#isRapid}}Rapid{{/isRapid}}ClientConfiguration = {{#isRapid}}Rapid{{/isRapid}}ClientConfiguration.EMPTY) : {{#isRapid}}RapidOpenApiStub{{/isRapid}}{{^isRapid}}OpenApiStub{{/isRapid}}(clientConfiguration) {
```

Comment:
This is a `deprecation` of the constructor, can you please keep the constructor `public` and add the `deprecation` annotation to it, and open a ticket to remove it in one of the upcoming releases?